src/main/java/org/example/Player.java

```diff
@@ -0,0 +1,6 @@
+package org.example;
+
+public abstract class Player {
+
+    public abstract char[][] play(char[][] matrix, char yourChar);
```

Comment:
when I read this method, name looks okay, matrix looks okay too, but yourChar parameter name does not tell me anything, maybe it could be named better.